### ISSUE TITLE
Road generation bug

### ISSUE THREAD

**Issue by Felokh** [IntellectualSites/PlotSquared#1819]
# Bug report template 

**Debug paste link:** 
https://hastebin.com/miyomorifo

**Description of the problem:** 
I want the roads to look like this: https://imgur.com/a/0xesC
When i do /plots debugroadregen they look like this: https://imgur.com/a/ES9wc
And they generate like this: https://imgur.com/a/PCXQ7
I use Spigot 1.12.2

**How to replicate:** 
Build roads and let them generate in the world.

**Checklist**:

- [x] I included a `/plot debugpaste` link
- [X] I made sure there are no duplicates of this report [(Use Search)](https://github.com/IntellectualSites/PlotSquared/issues?utf8=%E2%9C%93&q=is%3Aissue)
- [X] I made sure I am using an up-to-date version of PlotSquared
- [] I Made sure the bug/error is not caused by any other plugin

**Comment by lurr** [comment]
Roads in PlotSquared are generated in a different basis than here.
You create a corner piece, a horizontal piece, and a vertical piece.
As seen in [this image](https://i.imgur.com/Z9GxBub.jpg), you can see it operates on a grid pattern.

The lines in generation are a result of quartz in your schematics; expanding out your road design just a block wider on each axis would solve the generation issue once you regenerate your design.

**TL;DR:** Make your design a block longer each way, and then remake your template and regenerate :)
